rssfeed/templatetags/rssfeed_tags.py

```diff
@@ -7,5 +7,5 @@
 
 @register.inclusion_tag("rssfeed/inclusion_tags/render_rssfeed.html")
 def render_rssfeed(count=5):
-    entries = Entry.objects.all()[:count]
+    entries = Entry.objects.all()[:count].prefetch_related("feed")
```

Comment:
It doesn't matter where the select_related goes, django ORM is smart enough that it doesn't matter.